sdk/webpubsub/azure-messaging-webpubsub/src/main/java/com/azure/messaging/webpubsub/implementation/WebPubSubUtil.java

```diff
@@ -33,6 +33,8 @@ public static String getToken(BinaryData binaryData) {
                     String fieldName = parser.getCurrentName();
                     System.out.println(fieldName);
                     if (TOKEN.equals(fieldName)) {
+                        // move parser forward to get value of "token"
+                        parser.nextToken();
```

Comment:
Currently, when calling `getToken()`, we will get the name "token".
The token is wrong, it is: `wss://webpubsubhaoling.webpubsub.azure.com/client/hubs/test?access_token=token`

Reply:
Could you give a sample JSON, and what's previous "token", and what's fixed one?

Reply:
The input param `BinaryData` is a json like this:
```
{"token": aserqwtwertq...}
```
Here when we find the `fieldName.equals("token")`, the parser cursor is still pointing to the `token` key, we need to move the cursor forward by calling `parser.nextToken()` to let it point to the value of `token`